game/enemies/EnemyBuilding.gd

```diff
@@ -17,6 +30,7 @@ func populate() -> void:
 	
 	for node in get_random_spawn_points(soldier_count):
 		var soldier := SoldierScene.instantiate() as Soldier
+		soldier.setup(sfx_shoot, sfx_reload, sfx_death)
```

Comment:
why not preloading the sfx directly in the Soldier scene?

Reply:
because there are n soldiers and I was not able to find solid evidence on how preload works, I was aiming to ensure the same asset isn't loaded n times

Reply:
moved preload of these to soldier scene